core/rend/gles/gles.cpp

```diff
@@ -1964,61 +1979,81 @@ bool RenderFrame()
 
 		ShaderUniforms.Set(s);
 	}
+
 	//setup render target first
 	if (is_rtt)
 	{
-		GLuint channels,format;
+		const char * renderer = (char *)glGetString(GL_RENDERER);
```

Comment:
Wouldn't this be better in gl_init? Then we can do like `gl.workarounds.<whatever>`

Reply:
Yes, sure I will move it to gl_init